src/components/HURUmap/PanelButtonGroup/index.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import PanelButtonGroup from ".";
+
+import pinIcon from "@/pesayetu/assets/Component122.svg";
+import locationIcon from "@/pesayetu/assets/icons/location-pin.svg";
+import pinBlack from "@/pesayetu/assets/icons/office-push-pin.svg";
```

Comment:
1. Where do this icons come from? Why not use the ones in the [XD](https://xd.adobe.com/view/c00dbbc6-0de1-476d-bd2a-dd778c185a3a-955e/specs/)?
2. For images, lets stick to using the same names as in the XDs. Otherwise everyone is going to give images their own names leading to duplications.